aries-site/src/pages/templates/wizard.mdx

```diff
@@ -73,6 +73,24 @@ If guidance can be provided in a short description, use a single column wizard.
 - On desktop, place the guidance to the right of form inputs.
 - On mobile, place the guidance above form inputs.
 
+## Alignment
+
```

Comment:
There has also been confusion to where the "next button" should be aligned when content between steps is different widths. This came up in a recent office hours.

I believe the first example says "keep the wizard width consistent between steps so that the next button placement does not change between each step" or something along those lines. Can we address that guidance here to make it more clear?

Reply:
Include this in the step footer description of anatomy diagram.